src/content/docs/tips/astro-font.mdx

```diff
@@ -0,0 +1,160 @@
+
+<Steps>
+1. Make sure you have the local font in `public/fonts` directory. This example will use IBM Plex Sans
+  
+2. Configure `astro-font` in `layout.astro`
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
+          name: "IBMPlexSans",
+          src: [
+            {
+              style: 'normal',
+              weight: '400',
+              path: './public/fonts/IBMPlexSans-Regular.ttf'
+            },
+          ],
+          preload: true,
+          display: "swap",
+          fallback: "sans-serif",
+          cssVariable: "font-ibm"
+        }
+      ]}
+    />
+   ```
+
+3. Start using the font
+  
+   ```css title="globals.css"
+   body {
+       font-family: "font-ibm"
+   }
+   ```
+  
+</Steps>
+
+## With Google Fonts
+
+<Steps> 
+1. Configure `astro-font` in `layout.astro`. This example will use Inter font
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
+          src: [],
+          name: "Inter",
+          preload: false,
+          display: "swap",
+          fallback: "sans-serif",
+          cssVariable: "font-inter",
+          googleFontsURL: 'https://fonts.googleapis.com/css?family=Inter',
+        },
+      ]}
+    />
+   ```
+
+2. Start using the font
+  
+   ```css title="globals.css"
+   body {
+       font-family: "font-inter"
+   }
+   ```
+  
+</Steps>
+
+## With TailwindCSS
+
+<Steps> 
+1. Configure `astro-font` in `layout.astro`. This example will use Eduns font
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
+          name: "Eduns",
+          src: [
+            {
+              style: 'normal',
+              path: 'https://fonts.gstatic.com/s/edunswactfoundation/v2/raxRHjqJtsNBFUi8WO0vUBgc9D-2lV_oQdCAYlt_QTQ0vUxJki9tosGMci4.woff2'
+            },
+          ],
+          preload: true,
+          display: "swap",
+          fallback: "sans-serif",
+          cssVariable: "font-eduns",
+        },
+      ]}
+    />
+   ```
+
+2. Update tailwind config file
+  
+   ```js title="tailwind.config.mjs" ins={1,5-7}
+    import defaultTheme from 'tailwindcss/defaultTheme';
+    export default {
+      theme: {
+        extend: {
+          fontFamily: {
+            sans: ["var(--font-eduns)", ...defaultTheme.fontFamily.sans],
+          }
+        },
+      },
+      plugins: [],
+    }
+   ```
+
+3. Start using the font
+  
+   ```astro title="index.astro" ins="font-sans"
+      <p class="bg-blue-950 font-sans">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla tempus lacinia luctus.
+      </p>
+   ```
+  
+</Steps>
+
+:::note
+Make sure to read through [astro-font README](https://github.com/rishi-raj-jain/astro-font/blob/master/packages/astro-font/README.md) incase your deployment service does not supports Node.js runtime
```

Comment:
```suggestion
Make sure to read through [astro-font README](https://github.com/rishi-raj-jain/astro-font/blob/master/packages/astro-font/README.md) in case your deployment service does not supports Node.js runtime
```